### PR TITLE
Added new haiku named 2022_nevena_haiku

### PR DESCRIPTION
**Pull request by nverbic**
Added  a new haiku as part of the event " Introduction to Hacktoberfest 2022 and Open Source Contributions" organized by Women Who Code Python community.

### CODE REVIEW

File: _haikus/2022_nevena_haiku.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: 2022 nevena haiku
+author: Nevena Verbic
+---
+
+Programming languages.<br>
```
Comment (ltagliaferri) [inline _haikus/2022_nevena_haiku.md:7]:
```suggestion
Programming language.<br>
```

File: _haikus/2022_nevena_haiku.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Can we code a better world<br>
+together, in the clouds?<br>
```
Comment (ltagliaferri) [inline _haikus/2022_nevena_haiku.md:9]:
```suggestion
together, in clouds?<br>
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Thank you for coming to the Women Who Code event! It looks like you need to remove a syllable each from the first line and the last line. I made a suggestion, can you re-commit with an update so I can merge? Thank you!

**Comment by nverbic** [comment]
Hi, thank you for your review. I have commited your suggestions. I am not sure whether this is enough for you to merge the changes into the main branch. I am also a little bit worried about the "All checks have failed" (@github-actions
PR tests / test (pull_request_target) Failing after 12s)  notification that I can see under this pull request.  Should I run some tests before performing a pull request? 

Thank you!
Br
Nevena

**Review by ltagliaferri — APPROVED** [review]
Thank you for this haiku and for contributing to open source 🎉

**Pull request merged**